### PR TITLE
Integrate Availability Zone from private repository

### PR DESCRIPTION
**Pull request by yugangw-msft**
Mark as do not merge as I need 1 more change to use official compute and network sdks
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,19 +22,3 @@ jobs:
...
-      env: PURPOSE='Integration'
-      python: 3.6
-    - stage: verify
-      script: ./scripts/ci/test_integration.sh
-      env: PURPOSE='Integration'
-      python: 2.7
-    - stage: publish
-      script: ./scripts/ci/publish.sh
-      python: 3.6
-      if: branch = master and type = push
```
Comment (derekbekoe) [inline .travis.yml]: I think this is still needed. @troydai?

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ class ResourceType(Enum):  # pylint: disable=too-few-public-methods
 
     MGMT_STORAGE = ('azure.mgmt.storage',
                     'StorageManagementClient')
-    MGMT_COMPUTE = ('azure.mgmt.compute.compute',
+    MGMT_COMPUTE = ('azure.mgmt.compute',
```
Comment (derekbekoe) [inline src/azure-cli-core/azure/cli/core/profiles/_shared.py]: With the new SDK, is `azure.mgmt.compute` still correct?

File: src/command_modules/azure-cli-network/HISTORY.rst [hunk 3]
```diff
@@ -29,6 +29,7 @@ unreleased
 
 2.0.12 (2017-08-11)
 +++++++++++++++++++
+* `public-ip`: Add availability zone support.
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-network/HISTORY.rst]: This should be in `unreleased` I believe.

File: src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py [hunk 4]
```diff
@@ -908,6 +908,12 @@
         - name: Create a load balancer on a specific virtual network and subnet.
           text: >
             az network lb create -g MyResourceGroup -n MyLb --vnet-name MyVnet --subnet MySubnet
+        - name: create a zone flavored public facing load balancer through provisiong a zonal public ip
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py]: Typo: `provisiong`

File: test_clouds/brazilus.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
+	"Name": "ARMTest",
+	"EndpointActiveDirectoryResourceId": "https://management.core.windows.net/",
+	"EndpointGallery": "https://gallery.azure.com/",
+	"EndpointManagement": "https://brazilus.management.azure.com/",
+	"EndpointResourceManager": "https://brazilus.management.azure.com/",
+	"EndpointActiveDirectory": "https://login.windows.net/",
+	"EndpointActiveDirectoryGraphWindowsId": "https://graph.windows.net/",
+	"EndpointVmImageAliasDoc": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/quickstart-templates/aliases.json"
+}
```
Comment (derekbekoe) [inline test_clouds/brazilus.json]: Still keep this in public? @tjprescott?

File: .travis.yml (hunk 1 above)
Comment (yugangw-msft) [reply .travis.yml]: @derekbekoe, I will undo this when SDK gets published
https://github.com/Azure/azure-cli-pr/issues/13

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py (hunk 2 above)
Comment (yugangw-msft) [reply src/azure-cli-core/azure/cli/core/profiles/_shared.py]: yes

File: test_clouds/brazilus.json (hunk 5 above)
Comment (tjprescott) [reply test_clouds/brazilus.json]: No no. This should only be in the private repo.

File: src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py (hunk 4 above)
Comment (tjprescott) [reply src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py]: Zone flavored?  What does that taste like? 🍨

Comment (yugangw-msft) [reply src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py]: We don't want to say `zonal lb` which is conceptual wrong, but LB does have some relationships with zone. The `flavor` is the word I picked when I chatted with NRP folks.

File: packaged_releases/windows/scripts/build_local.cmd
Comment (jiasli) [inline packaged_releases/windows/scripts/build_local.cmd:1]: This PR does more than "Integrate Availability Zone from private repository". It also introduces `packaged_releases/windows/scripts/build_local.cmd` which is very similar to `packaged_releases/windows/scripts/prepareBuild.cmd`. `packaged_releases/windows/scripts/build_local.cmd` should have been moved to a separate PR.

Comment (jiasli) [inline packaged_releases/windows/scripts/build_local.cmd:1]: This PR does more than "Integrate Availability Zone from private repository". It also introduces `packaged_releases/windows/scripts/build_local.cmd` which is very similar to `packaged_releases/windows/scripts/prepareBuild.cmd` (introduced by #2655). `packaged_releases/windows/scripts/build_local.cmd` should have been moved to a separate PR.

Later in #4696, `packaged_releases/windows/scripts/build_local.cmd` was renamed to `build_scripts/windows/scripts/build.cmd` and `packaged_releases/windows/scripts/prepareBuild.cmd` was deleted.
